api/app/Models/PoolCandidate.php

```diff
@@ -761,6 +761,25 @@ private function addSkillCountSelect(Builder $query, ?array $skills): Builder
         ]);
     }
 
+    /**
+     * Custom sort to handle issues with how laravel aliases
+     * aggregate selects and orderBys for json fields in `lighthouse-php`
+     *
+     * The column used in the orderBy is `table_aggregate_column->property`
+     * But is actually aliased to snake case `table_aggregate_columnproperty`
+     */
+    public function scopeOrderByPoolName(Builder $query, ?array $args): Builder
+    {
+        extract($args);
```

Comment:
Not a logic concern but a naming one, scopes are used to refer to filtering right? In this case, it isn't really a scope or doing any scoping in the Laravel sense of the word. Not a big deal but just thinking about it

Reply:
Scopes are just for query constraints in general. Imo, while it is a little odd to do ordering in a scope, it can be considered a constraint and be applicable.